home page/stylesheet.css

```diff
@@ -1,35 +1,46 @@
 @import url('https://fonts.googleapis.com/css2?family=Irish+Grover&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Gravitas+One&family=Irish+Grover&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Graduate&family=Gravitas+One&family=Irish+Grover&display=swap');
+*{
+  padding: 0;
+  margin: 0;
+}
+:root{
+  --main-btn-color: #2F2F2FCC;
```

Comment:
The background color of the footer in the page is not the same as what was set in this styling. Check the Figma design to match color.